tests/airbender/basic/src/airbender_rt.rs

```diff
@@ -0,0 +1,60 @@
+use core::alloc::{GlobalAlloc, Layout};
```

Comment:
Huh, I see -- I wonder why runtime stuff this isn't part of some crate in their SDK?

Reply:
I think their guest SDK is very minimum, they don't even provide a default working allocator (their default is a panicing one), so here we have enable the feature `custom_allocator`, then implement our own one.

Their SDK mainly provides read input words, and commit output 32 bytes.